app/com/linkedin/drelephant/spark/heuristics/StagesHeuristic.scala

```diff
@@ -147,7 +174,17 @@ object StagesHeuristic {
       stagesAndAverageExecutorRuntimeSeverities
         .collect { case (stageData, runtime, severity) if severity.getValue > Severity.MODERATE.getValue => (stageData, runtime) }
 
-    lazy val severity: Severity = Severity.max((stageFailureRateSeverity +: (taskFailureRateSeverities ++ runtimeSeverities)): _*)
+    lazy val severity: Severity = Severity.max((stageFailureRateSeverity +: severityTimeA +: severityTimeD +: (taskFailureRateSeverities ++ runtimeSeverities)): _*)
+
+    var (jvmTime, executorCpuTime) = getTimeValues(stageDatas)
+
+    var ratio: Double = {
+      ratio = jvmTime.toDouble / executorCpuTime.toDouble
```

Comment:
Jvm GC time is in wall clock, and Executor CPU time is in CPU clock. I don't know if dividing them is right to do here. Instead of using executor CPU time, we should executor duration.